src/test/java/rife/engine/ContextInfo.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 Erik C. Thauvin (https://erik.thauvin.net/)
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ */
+package rife.engine;
+
+public class ContextInfo {
```

Comment:
I don't understand the user of this class, it seems to just store the information from `Context` and that is then used to access it again. Why not simply use the context methods directly?

Reply:
Me neither, maybe I was thinking I was going to reuse it somewhere else, or something.

Reply:
Well, tests failed. I remembered that I did it that way because it is set in a lambda and must be a final object.